google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeployAction.java

```diff
@@ -167,7 +155,20 @@ public void onTextAvailable(ProcessEvent event, Key outputType) {
     public void processTerminated(final ProcessEvent event) {
       try {
         if (event.getExitCode() == 0) {
-          callback.succeeded(new DeploymentRuntimeImpl(deploymentOutput.toString(), version));
+
+          // Parse JSON output to retrieve service/version info.
+          DeployOutput deployOutput = null;
+          try {
+            deployOutput = parseDeployOutput(deploymentOutput.toString());
+          } catch (JsonParseException e) {
+            getLoggingHandler().print(
```

Comment:
For both `consoleLogLn()` and `logger.error()`, they actually don't need text localization. However, not using `GctBundle.message()` results in embedding a very long string in the code such as below. It will also make a bit difficult to maintain consistent messaging if it's outside the resrouce bundle.

"Failed to parse service/version info from the ''gcloud'' deployment output. \nPlease make sure that you are using the latest version of the Google Cloud SDK.\nRun ''gcloud components update'' to update the SDK. (See: https://cloud.google.com/sdk/gcloud/reference/components/update.)"

Suggestions?